src/graphics/program-lib/chunks/normalVertex.frag

```diff
@@ -1,4 +1,6 @@
 void getNormal() {
     dNormalW = normalize(dVertexNormalW);
-}
-
+    #ifdef CLEARCOAT
+        ccNormalW = normalize(dVertexNormalW);
+    #endif
+}
```

Comment:
Also, remember that these chunks are glued together and the whitespace matters because it helps legibility when you're debugging GLSL. There used to be a newline at the end of this file so I'd recommend restoring that.

Reply:
restored (was 2 lines)